docs/calls.md

```diff
@@ -1,17 +1,21 @@
 `apimetrics calls`
 ==================
 
-Manage monitoring calls.
+Manage monitoring calls, which can be separated into two sections:
 
-* [`apimetrics calls`](#apimetrics-calls)
-* [`apimetrics calls create`](#apimetrics-calls-create)
-* [`apimetrics calls edit`](#apimetrics-calls-edit)
-* [`apimetrics calls schedules add`](#apimetrics-calls-schedules-add)
-* [`apimetrics calls schedules delete`](#apimetrics-calls-schedules-delete)
+* [API Calls](#api-calls)
+  * [`apimetrics calls`](#apimetrics-calls)
+  * [`apimetrics calls create`](#apimetrics-calls-create)
+  * [`apimetrics calls edit`](#apimetrics-calls-edit)
+* [Schedules](#schedules)
+  * [`apimetrics calls schedules add`](#apimetrics-calls-schedules-add)
+  * [`apimetrics calls schedules delete`](#apimetrics-calls-schedules-delete)
```

Comment:
These lists are generated automatically so that when we add a new command, it is automatically updated. I don't think we can add formatting here